lib/trino-filesystem/src/test/java/io/trino/filesystem/AbstractTestTrinoFileSystem.java

```diff
@@ -63,11 +71,22 @@ public abstract class AbstractTestTrinoFileSystem
 
     protected abstract void verifyFileSystemIsEmpty();
 
-    protected boolean supportsCreateExclusive()
+    /**
+     * Specifies whether implementation supports {@link TrinoOutputFile#create()} is exclusive.
+     */
+    protected boolean isCreateExclusive()
```

Comment:
these two methods are pretty close in naming which can be rather misleading.

Reply:
that's why they got a javadoc. do you want to suggest better names?